examples/menu/src/App.tsx

```diff
@@ -0,0 +1,39 @@
+import { Typography } from "@material-ui/core";
+import { CalendarToday, Home, School, Settings } from "@material-ui/icons";
+import { Master, Menu, MenuCollapsibleItem, MenuItemLink } from "@vivid-planet/react-admin-mui";
+import * as React from "react";
+import { Route, Switch } from "react-router";
+
+function AppMenu() {
+    return (
+        <Menu>
+            <MenuItemLink text="Foo1" icon={<CalendarToday />} path="/foo1" />
+            <MenuItemLink text="Foo2" icon={<School />} path="/foo2" />
+            <MenuCollapsibleItem text="Foo3" icon={<Settings />} collapsible={true} isOpen={false}>
+                <MenuItemLink text="Foo4" icon={<Home />} path="/foo4" />
+                <MenuItemLink text="Foo5" icon={<Home />} path="/foo5" />
+            </MenuCollapsibleItem>
+        </Menu>
+    );
+}
+
+export default function App() {
+    return (
+        <Master
+            renderHeader={() => (
+                <Typography variant="h5" color="primary">
+                    Example
+                </Typography>
+            )}
+            menuComponent={AppMenu}
```

Comment:
renderHeader vs menuComponent:
- should we support renderHeader and headerComponent + renderMenu and menuComponent?
- should we only use renderHeader and renderMenu
- or headerComponent and menuComponent

Reply:
I have no preference here, but I would not support both.

Reply:
headerComponent and menuComponent

Reply:
What do other libraries use?

Reply:
headerComponent and menuComponent